src/Uno.Extensions.Reactive.UI.Tests/Presentation/Bindings/Collections/Given_BindableListFeed.cs

```diff
@@ -40,4 +40,21 @@ public async Task When_GetSourceWithDispatcherInSourceContext_Then_GetCollection
 		async Task<object?> GetBindableCollection(SourceContext context)
 			=> (await ((ISignal<IMessage>)sut!).GetSource(context).FirstAsync(CT)).Current.Data.SomeOrDefault();
 	}
+
+	[TestMethod]
+	public async Task When_CollectionChanged_Then_CountPropertyChangedRaised()
+	{
+		var vm = new BindableGiven_BindableListFeed_Model();
+		var sut = vm.Items as BindableListFeed<int>;
+
+		sut.Should().NotBeNull();
+
+		var countHasChanged = false;
+		sut!.PropertyChanged += (snd, e) => countHasChanged |= e.PropertyName is null or { Length: 0 } or "Count";
```

Comment:
Why is coundHasChanged being set to true when the PropertyName is null or empty?

Reply:
Raising a property changed with `PropertyName` being `null` or `string.Empty` (depending of the platform) means that "all properties has changed" :

"Raising the PropertyChanged event with an argument of [String.Empty](https://learn.microsoft.com/en-us/dotnet/api/system.string.empty) or null indicates that all non-indexer properties on the object should be re-read." https://learn.microsoft.com/en-us/windows/uwp/data-binding/data-binding-in-depth#binding-source